src/app/static/src/app/components/projects/ShareProject.vue

```diff
@@ -119,6 +122,13 @@
         methods: {
             cloneProject: mapActionByName("projects", "cloneProject"),
             userExists: mapActionByName("projects", "userExists"),
+            enterEmails(email: EmailToShareWith, index: number){
+                this.addEmail(email, index)
+                let self = this
```

Comment:
I think this is the cause of build failure - you can maybe just do this:
```
this.cycleInputs(email, index)
```
because the arrow function should bind `this` appropriately.